content/posts/2017/2017-10-17-plotly_vs_ggplotly.Rmd

```diff
@@ -0,0 +1,121 @@
+---
+title: "plotly vs ggplot2"
+author: "Theo Roe"
+date: 2017-10-17
+output: html_document
+draft: true
+slug: "plotly_vs_ggplotly"
+tags: [r,tidyverse, graphics, ggplot2, plotly]
+---
+
+```{r setup, include=FALSE}
+knitr::opts_chunk$set(echo = TRUE, collapse = TRUE)
+library(mosaicData)
```

Comment:
Where ever possible, I try to avoid hidden packages. It's too easy to miss them out in the text. In the code below, separate out the install.packages into one chunk and the library calls to another. For example

We need 
```{r, eval=FALSE}
install.packages(c("A", "B", "C"))
```
which are loaded in the usual way
```{r}
library("a")
```